akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -36,8 +36,6 @@ import akka.util.PrettyDuration._
 
 @InternalApi
 private[akka] object TestProbeImpl {
-  private val testActorId = new AtomicInteger(0)
```

Comment:
I think that import is too magic/hidden. Would prefer 
```
// have to use same global counter as Classic TestKit to ensure unique names
private def testActorId = akka.testkit.TestKit.testActorId
```

Reply:
Okay